### ISSUE TITLE
How to use the output of the graph as a parameter of the loss function

### ISSUE THREAD

**Issue by jren2017** [tensorflow/tensorflow#13373]
Is there any example about using the output of the graph as a parameter of the loss function?
For example:

_,_,summary_str_train, **acc_train** = sess.run([train_step, cross_entropy, merged_summary_op, accuracy], feed_dict={x: batch_xs, y: batch_ys, keep_prob: 0.7})

acc_train is the accuracy of the current minibatch, output of the graph, I want to use the value of the acc_train in the loss function? the cross_entropy_loss is a customized function. 

cross_entropy = tf.reduce_mean(cross_entropy_loss(logits=prediction, one_hot_labels=y, accuracy=acc_train))

Is there any example about this? Thanks

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
